.gitlab-ci.yml

```diff
@@ -275,6 +271,25 @@ build-linux-subkey:
     - cp -r scripts/docker/subkey.Dockerfile ./artifacts/subkey/
     - sccache -s
 
+build-full-crypto-feature:         
```

Comment:
You should copy from the `srml-staking` test. This is not correct. You are not using the correct stage etc. Please compare with the `srml-staking` test and also move this test above line 223.

Reply:
thought it should be build stage because it's build, not test.
I had that before but then I got errors. Let's see what my last commit does

Reply:
done